core/variant/variant.h

```diff
@@ -204,7 +204,9 @@ class Variant {
 		PackedArrayRefBase *packed_array;
 		void *_ptr; //generic pointer
 		uint8_t _mem[sizeof(ObjData) > (sizeof(real_t) * 4) ? sizeof(ObjData) : (sizeof(real_t) * 4)];
-	} _data alignas(8);
+	} _data {
+	}
+	alignas(8);
```

Comment:
Aside from the bogus style changes from `clang-format`, I assume that this is *not* something that we want to do? I'm not even sure how that would work, I'm having trouble finding any proper documentation on how unions should or can be initialized.